crates/bevy_anti_aliasing/src/dlss/mod.rs

```diff
@@ -42,52 +44,118 @@ use dlss_wgpu::{
         DlssRayReconstruction, DlssRayReconstructionDepthMode, DlssRayReconstructionRoughnessMode,
     },
     super_resolution::DlssSuperResolution,
+    FeatureSupport,
 };
 use std::{
     marker::PhantomData,
     ops::Deref,
     sync::{Arc, Mutex},
 };
 use tracing::info;
+use uuid::Uuid;
 
-pub use bevy_render::{
-    DlssProjectId, DlssRayReconstructionSupported, DlssSuperResolutionSupported,
-};
-pub use dlss_wgpu::DlssPerfQualityMode;
+/// Initializes DLSS support in the renderer. This must be registered before [`RenderPlugin`](bevy_render::RenderPlugin) because
+/// it configures render init code.
+#[derive(Default)]
+pub struct DlssInitPlugin;
+
+impl Plugin for DlssInitPlugin {
+    fn build(&self, app: &mut App) {
+        let dlss_project_id = app.world().get_resource::<DlssProjectId>()
+                        .expect("The `dlss` feature is enabled, but DlssProjectId was not added to the App before DlssPlugin.").0;
```

Comment:
This should say DlssInitPlugin, technically.

But in practice for most users this will be DefaultPlugins.